apps/projects/test/Projects.test.js

```diff
@@ -1,4 +1,4 @@
-/* global artifacts, assert, before, beforeEach, contract, context, expect, it, web3 */
+/* global artifacts, assert, before, beforeEach, contract, context, expect, it, web3, xit */
```

Comment:
`xit` should actually not need to be defined here, as tests should only be skipped during deployment